xcube_cds/datasets/satellite_soil_moisture.py

```diff
@@ -146,14 +154,11 @@ def get_open_data_params_schema(self, data_id: str) -> JsonObjectSchema:
             crs=JsonStringSchema(nullable=True, default='WGS84',
                                  enum=[None, 'WGS84']),
             # W, S, E, N (will be converted to N, W, S, E).
-            # For the soil moisture dataset, all data is global and no
-            # geographic subsetting is possible, so the values are fixed
-            # (i.e. minimum == maximum for every limit).
             bbox=JsonArraySchema(items=(
-                JsonNumberSchema(minimum=-180, maximum=-180),
-                JsonNumberSchema(minimum=-90, maximum=-90),
-                JsonNumberSchema(minimum=180, maximum=180),
-                JsonNumberSchema(minimum=90, maximum=90))),
+                JsonNumberSchema(minimum=-180, maximum=180, multiple_of=0.25),
+                JsonNumberSchema(minimum=-90, maximum=90, multiple_of=0.25),
+                JsonNumberSchema(minimum=-180, maximum=180, multiple_of=0.25),
+                JsonNumberSchema(minimum=-90, maximum=90, multiple_of=0.25))),
```

Comment:
This is neither handled by the gen UI nor can a user easily know that bboxes must be given as `multiple_of=0.25`. Subsetting will take care automaticlly by snapping to the 0,.25 degree grid.

```suggestion
                JsonNumberSchema(minimum=-180, maximum=180),
                JsonNumberSchema(minimum=-90, maximum=90),
                JsonNumberSchema(minimum=-180, maximum=180),
                JsonNumberSchema(minimum=-90, maximum=90))),
```

Reply:
Fixed in commit ec8d394.